src/OpenTelemetry/Metrics/MetricReaderExt.cs

```diff
@@ -137,45 +125,39 @@ internal List<Metric> AddMetricsListWithViews(Instrument instrument, List<Metric
 
                     if (this.instrumentIdentityToMetric.TryGetValue(metricStreamIdentity, out var existingMetric))
                     {
-                        // The list of metrics may already contain a matching metric with the same
-                        // identity when a single instrument is selected by multiple views.
-                        if (!metrics.Contains(existingMetric))
-                        {
-                            metrics.Add(existingMetric);
-                        }
-
+                        metrics.Add(existingMetric);
```

Comment:
The conditional here is no longer required. Its purpose was to "merge" together identical streams resulting from different views. This primary purpose of this PR is to _not_ do this merging.